_components/search-bar.md

```diff
@@ -64,7 +64,7 @@ title: Search Bar
     <h3>Use</h3>
     <ul>
       <li>Allow the search bar to be as wide as possible, but a minimum of 27 characters wide. This allows users to enter multiple search terms without their inputs being obscured. The more users can see their search terms, the easier it is to review, verify, and submit their search query.</li>
-      <li>The magnifying glass has been shown to be almost universally recognized by users as an indicator of search functionality, and doesn’t need to be paired with the word "Search."</li>
+      <li>The magnifying glass has been shown to be almost universally recognized by users as an indicator of search functionality and doesn’t need to be paired with the word "Search" (except for screen readers, see the accessibility guidelines).</li>
```

Comment:
Suggestion for word-smithing to be shorter: 
"The magnifying glass is almost universally understood and doesn’t need to be visually paired with the word "Search." However, be sure to provide text for screen readers, as noted under Accessibility."